package.json

```diff
@@ -34,8 +34,13 @@
     "nodemon": "^2.0.7"
   },
   "dependencies": {
+    "@fast-csv/parse": "^4.3.6",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "fast-csv": "^4.3.6",
+    "node-postgres": "^0.6.2",
+    "pg": "^8.5.1",
+    "readline-promise": "^1.0.5"
```

Comment:
what is this package? and what is fast-csv?

Reply:
they're packages I was using to import the csv. I will be removing them in future PRs.